### DISCUSSION TITLE
Proposal for a Private Option for the Following Feature on Profile

### DISCUSSION THREAD

**Discussion by Loganloxsins** [community/community#54550] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Dear GitHub Team,

I would like to propose the addition of a "Private" option for the "Following" feature on GitHub.

Currently, the "Following" list is visible to the public, which can lead to privacy concerns for some users. By adding a "Private" option, users would have the ability to keep their "Following" list visible only to themselves, providing an extra layer of privacy and security.

I believe that this option would be a valuable addition to the platform, as it would allow users to better control their personal information and activity on the site. Additionally, this would align with the privacy features already available on GitHub, such as the ability to make repositories private and limit the visibility of personal information.

Thank you for your consideration of this suggestion. I hope that you will take this feedback into account as you continue to improve and develop the GitHub platform.

Sincerely,
Loganloxsins.

**Comment by Loganloxsins** [comment]
I would greatly appreciate having this feature in the near future!Thank uuuu!!!

**Comment by wzjoriv** [comment]
The new private profile option is a good start but I don't want to make everything private. I would prefer if we can select what to make private like the following list as mentioned here but not my pinned repositories for example.
